app/models/container.rb

```diff
@@ -41,8 +41,9 @@ def event_where_clause(assoc = :ems_events)
 
   PERF_ROLLUP_CHILDREN = []
 
-  def perf_rollup_parents(_interval_name = nil)
+  def perf_rollup_parents(interval_name = nil)
     # No rollups: nodes performance are collected separately
```

Comment:
I don't this comment applies anymore

Reply:
Maybe leave a comment about not rolling up to the nodes?  But yeah `No rollups` is not correct anymore